[bug] `guacone query bad` failures with no `CertifyBad` nodes

**Describe the bug**

When using `guacone query bad` with no `CertifyBad` nodes in the graph, the CLI offers no clean exit and will sometimes crash on interaction:

```
panic: runtime error: index out of range [-1]

goroutine 65 [running]:
github.com/manifoldco/promptui/list.(*List).Items(...)
	/home/runner/go/pkg/mod/github.com/manifoldco/promptui@v0.9.0/list/list.go:233
github.com/manifoldco/promptui.(*Select).innerRun.func1({0x10577f440?, 0x0?, 0x0?}, 0x1026fc280?, 0x77b030?)
	/home/runner/go/pkg/mod/github.com/manifoldco/promptui@v0.9.0/select.go:309 +0xce4
github.com/chzyer/readline.(*DumpListener).OnChange(0x1400077afa0?, {0x10577f440?, 0x1028c50d6?, 0x2?}, 0x140012ebf60?, 0x1?)
	/home/runner/go/pkg/mod/github.com/chzyer/readline@v1.5.1/operation.go:522 +0x3c
github.com/chzyer/readline.(*Operation).ioloop(0x1400072c1c0)
	/home/runner/go/pkg/mod/github.com/chzyer/readline@v1.5.1/operation.go:341 +0x1460
created by github.com/chzyer/readline.NewOperation in goroutine 1
	/home/runner/go/pkg/mod/github.com/chzyer/readline@v1.5.1/operation.go:88 +0x2c4
```

**To Reproduce**
Steps to reproduce the behavior:

1. Run `guacone query bad`
2. Mash the arrow keys a bit. I haven't been able to determine an exact number of arrow key presses that will trigger this.

**Expected behavior**

Two behaviors expected:

1. CLI prints `No results` and exits instead of waiting for interaction.
2. CLI does not panic

**GUAC version**
v0.12.0

--- a/cmd/guacone/cmd/bad.go
+++ b/cmd/guacone/cmd/bad.go
@@ -65,6 +65,11 @@ var queryBadCmd = &cobra.Command{
 			logger.Fatalf("error querying for package: %v", err)
 		}
 
+		if len(certifyBadResponse.CertifyBad) == 0 {
+			fmt.Println("No CertifyBad nodes found!")
+			return
+		}
+
 		mapCertifyBad := map[string][]model.CertifyBadCertifyBad{}
 		for _, certifyBad := range certifyBadResponse.CertifyBad {
 			switch subject := certifyBad.Subject.(type) {

